### PR TITLE
[Doc] Fix readme error link

### PR DESCRIPTION
**Pull request by stalary**
# Proposed changes

Issue Number: close #xxx

## Problem Summary:
Fix readme error link, such as Compilation and Basic Usage

## Checklist(Required)

1. Type of your changes:
    - [ ] Improvement
    - [x] Fix
    - [ ] Feature-WIP
    - [ ] Feature
    - [x] Doc
    - [ ] Refator
    - [ ] Others: 
2. Does it affect the original behavior: 
    - [ ] Yes
    - [x] No
    - [ ] I don't know
3. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [x] No Need
4. Has document been added or modified:
    - [x] Yes
    - [ ] No
    - [ ] No Need
5. Does it need to update dependencies:
    - [ ] Yes
    - [x] No
6. Are there any changes that cannot be rolled back:
    - [ ] Yes
    - [x] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by hf200012 — DISMISSED** [review]
LGTM

**Review by hf200012 — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -55,11 +55,11 @@ Doris mainly integrates the technology of [Google Mesa](https://research.google/
 
 ## Compile and install
 
-See [Compilation](https://doris.apache.org/docs/install/source-install/compilation-with-ldb-toolchain.html)
+See [Compilation](https://doris.apache.org/docs/install/source-install/compilation/)
```
Comment (adonis0147) [inline README.md:58]:
```suggestion
See [Compilation](https://doris.apache.org/docs/install/source-install/compilation-with-ldb-toolchain)
```

File: README.md [hunk 2]
```diff
@@ -55,11 +55,11 @@ Doris mainly integrates the technology of [Google Mesa](https://research.google/
...
 
 ## Getting start
 
-See [Basic Usage](https://doris.apache.org/docs/data-table/basic-usage.html)
+See [Basic Usage](https://doris.apache.org/docs/get-starting/)
```
Comment (adonis0147) [inline README.md:62]:
```suggestion
See [Basic Usage](https://doris.apache.org/docs/data-table/basic-usage)
```

**Review by adonis0147 — CHANGES_REQUESTED** [review]

**Comment by adonis0147** [comment]
Hi @stalary , the old links are broken due to the suffix `.html`. I think removing the suffix is better.

**Comment by stalary** [comment]
> Hi @stalary , the old links are broken due to the suffix `.html`. I think removing the suffix is better.

Ok, I will modify it

**Comment by stalary** [comment]
done. @adonis0147

**Review by adonis0147 — APPROVED** [review]
LGTM

**Comment by adonis0147** [comment]
Duplicated. See #11091

**Comment by stalary** [comment]
> Duplicated. See #11091

Thanks for reminding me, I will close this PR.

**Pull request closed without merging**